webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -171,9 +223,45 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
             <Form.Item
               label="Model name"
               name="modelName"
-              rules={[{ message: 'Model name is required ', required: true }]}>
+              required
+              requiredMessage="Model name is required ">
               <Input onChange={handleNameChange} />
             </Form.Item>
+            {isRbacEnabled && (
+              <Tooltip placement="bottom">
+                <Form.Item label="Workspaces" name="workspace">
+                  <Dropdown
+                    arrow
+                    className={css.workspacDropdown}
+                    disabled={!workspaces.length || isWorkspace}
+                    dropdownRender={(menu) =>
+                      React.cloneElement(menu as React.ReactElement, {
```

Comment:
according to [this official doc](https://beta.reactjs.org/reference/react/cloneElement), its safe to avoid using `cloneElement`